willow/app/assets/stylesheets/sufia.scss

```diff
@@ -2,6 +2,7 @@
  *= require jquery-ui
  *= require select2
  *= require_self
+ *= require_tree .
```

Comment:
I think its better to be explicit about which files are being included, rather than implicit.